ios/jailed/patchapp.sh

```diff
@@ -0,0 +1,377 @@
+#!/bin/bash
+
+#
+# You might need to change these
+#
+OPTOOL="/usr/local/bin/optool"
+IOSDEPLOY="/usr/local/bin/ios-deploy"
```

Comment:
Ideally they should already be in /usr/local/bin (e.g. with a `brew install ios-deploy`), or the user might add them to `$THEOS/bin` or another location in their $PATH. In other words just assume the user has already put it somewhere in their $PATH, and show an error if not. Having to enter the paths over and over when building would be irritating.